src/mmw/js/src/core/modals/models.js

```diff
@@ -30,8 +30,35 @@ var ShareModel = Backbone.Model.extend({
     }
 });
 
+var AlertTypes = {
+    info: {
+        alertHeader: 'Information',
+        alertIcon: '\"fa fa-info-circle\"',
+        dismissLabel: 'Okay'
+    },
+    warn: {
+        alertHeader: 'Warning',
+        alertIcon: '\"fa fa-exclamation-triangle\"',
+        dismissLabel: 'Okay'
+    },
+    error: {
+        alertHeader: 'Error',
+        alertIcon: '\"fa fa-ban\"',
```

Comment:
I didn't know that quoting `{{ alertIcon }}` put literal quotes in the markup, I kept wondering why it wasn't working so I escaped the strings in the model.